src/templates/default-alertmanager-config.yaml.tpl

```diff
@@ -15,6 +15,10 @@ inhibit_rules:
       severity: warning
     target_match_re:
       severity: info
+  # this rule inhibits ALL alerts during a clusterupdate,
+  # except the MCDDrainError that can indicate a blocked update
+  - source_matchers: [alertname = cluster-is-updating]
```

Comment:
Alertname bitte (siehe Kommentar in PR im gp-helm-chart-development) beim Namen ändern.